File: Python Word Examples/MailMergeApiDemo/mailMerge.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.mail_merge_api.MailMergeApi(configuration)

file_name = 'MailMergeFormField.doc'
merging_list = []
merging_list.append(MailMergeData('Contact Name', 'E-iceblue'))
merging_list.append(MailMergeData('Fax', '123456789'))
merging_list.append(MailMergeData('Date', '2019-10-16'))
dest_file_path = 'output/MailMerge.doc'
extra = {'folder': 'input'}
result = api.mail_merge(file_name, dest_file_path, merging_list, **extra)

File: PHP Word Examples/MailMergeApiDemo/mailMerge.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\MailMergeApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;
use Spire\Cloud\Word\Sdk\Client\Model\MailMergeData;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new MailMergeApi($configuration);

$name = "MailMerge.docx";
$folder = "input";
$storage = null;
$password = null;
$dest_file_path = "output/MailMerge.docx";
$data = array(
    new MailMergeData(array('name' => 'Contact Name', 'value' => 'E-iceblue')),
    new MailMergeData(array('name' => 'Fax', 'value' => '123456789')),
    new MailMergeData(array('name' => 'Date', 'value' => '2019-10-16'))
);
$result = $apiInstance->mailMerge($name, $dest_file_path, $data, $folder, $storage, $password);